public/caesar-shift/decrypt.py

```diff
@@ -0,0 +1,32 @@
+import string
+from encrypt_library import *
+from encrypt import caesar_shift
+#START#
+def caesar_shift_decrypt(ciphertext, shift_value=5):
+  #we can flip the shift value we think the encryptor used to decrypt the message!
+  shift_value = -shift_value
+    #<a href="#" class="clickable" onclick="if(enable_tooltips){$('#modalDecryptEncrypt').modal('open')}">return caesar_shift(ciphertext, shift_value)</a>
+  return caesar_shift(ciphertext, shift_value)
+#END#
+
+def test_decryption():
```

Comment:
As with `encrypt.py()`, I recommend finding a way to move the tests out, and decide what we want to do with `main()`.